components/core/src/clp_s/SchemaTree.hpp

```diff
@@ -93,7 +113,26 @@ class SchemaTree {
         return m_nodes[id];
     }
 
-    int32_t get_root_node_id() const { return m_nodes[0].get_id(); }
+    /**
+     * @return the Id of the root of the Object sub-tree.
```

Comment:
And can we also explain it somewhere that the object sub-tree stores log data (while the internal sub-tree stores metadata)

Reply:
Hmm, I don't agree with this one. Metadata side should be clear enough after renaming it from Internal -> Metadata. The object sub-tree stores object-like data which may or may not be log data. Also, since we plan to store heterogeneous logs in the future I don't want to imply in the language that it's the main class of log data or that it is even guaranteed to exist.

Reply:
Got it. When might object-like data not count as log data?

Reply:
E.g. if someone dumps an sqlite database as JSON and ingests that.

I can add something like "which is expected to contain JSON log data" to the docstring though, since that reflects the current use.